backup.html

```diff
@@ -56,17 +56,38 @@ <h2>Backup center</h2>
             <br><br>
 
             <div class="placeholder">BIP32 extended master private key</div>
-            <textarea id="backup-xpriv" class="text-input" rows="3" cols="70" wrap="hard" spellcheck="false" readonly></textarea>
+            <output id="backup-xpriv" class="extended-keys"></output>
```

Comment:
why did you chose to exchange textarea with output?

Reply:
Textarea seems to be used for displaying (static) text and allow user editing. Output on the other hand seems much more fitting, since it is the output of some calculation.
The reason I chose it though is that I think it renders more compactly with the current css, so it's just more convenient for the user, since there is less scrolling involved.

Reply:
I'm just a bit concerned about the browser support and would therefore recommend to use a div with styling, and possibly in another PR where all textareas are replaced.

Reply:
I like the point with div styling. Removed the output tag again and reverted to the original textarea.